dist/systemd/obsapidelayed-queue-consistency_check.service

```diff
@@ -0,0 +1,16 @@
+[Unit]
+Description = OBS API Delayed Queue consistency_check server
```

Comment:
This is not a server. I would suggest: `Open Build Service DelayedJob Queue: consistency_check` and the unit `obs-delayedjob-queue-consistency_check.service`.

Same for the other units of course...

Reply:
@hennevogel 

> This is not a server. I would suggest: Open Build Service DelayedJob Queue: consistency_check and the unit obs-delayedjob-queue-consistency_check.service.
>
> Same for the other units of course...

Changed all of them.